pages/privacy.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+* privacy-policy.tsx
+* @2023 Open Seattle
+*/
+
+import {
+    Box,
+    Container,
+    ListItemText,
+    Stack,
+    Typography,
+    useMediaQuery,
+    useTheme
+} from '@mui/material'
+import Masthead from 'components/Masthead'
+import { withBasicLayout } from 'components/layouts'
+import { ReactNode } from 'react'
+
+type CardCopyTextProps = {
+    title?: string,
+    children: ReactNode
+}
+const CardCopyText = ({ title, children }: CardCopyTextProps) => {
+    const isSmallScreen = useMediaQuery('(max-width:600px)')
```

Comment:
We cannot use breakpoint here?

Reply:
`const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'))`

Reply:
Likewise, i'll fix after merge.